Makefile

```diff
@@ -6,15 +6,17 @@ all:
 install:
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
 	mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
-	mkdir -p $(DESTDIR)/etc/neofetch
+	mkdir -p $(DESTDIR)$(PREFIX)/etc/neofetch
```

Comment:
Here's the weird thing, they used `$(PREFIX)` and not `$(DESTDIR)` to change their primary target.

`/the/actual/prefix/etc/neofetch` can be done with `make DESTDIR=/the/actual/prefix`. Most of the distros do this (Gentoo with `DESTDIR=/var/tmp/portage/neofetch-3.1.0/image` when they're testing their ebuild so they'll be put in `/var/tmp/portage/neofetch-3.1.0/image`) , so even if the PREFIX is `/usr/local`, if the DESTDIR is `/the/actual/prefix`, then the makefile will put it in `/the/actual/prefix/usr/local`.